pkg/test/framework/components/istio/config.go

```diff
@@ -153,6 +153,9 @@ type Config struct {
 
 	// EnableCNI indicates the test should have CNI enabled.
 	EnableCNI bool
+
+	// custom deployment for ingress and egress gateway
```

Comment:
Might be good to be clear that this only applies to gateways installed on remote clusters?
```suggestion
	// Custom deployment for ingress and egress gateways on remote clusters.
```